src/main/java/net/sf/jabref/logic/bibtex/LatexFieldFormatter.java

```diff
@@ -49,24 +49,24 @@
     private final FieldContentParser parser;
 
 
-    public LatexFieldFormatter() {
-        this(true);
+    public LatexFieldFormatter(JabRefPreferences prefs) {
+        this(true, prefs);
     }
 
-    private LatexFieldFormatter(boolean neverFailOnHashes) {
+    private LatexFieldFormatter(boolean neverFailOnHashes, JabRefPreferences prefs) {
```

Comment:
1. Makes sense. Passing an object may be a good idea, I agree.

2. Ahh, that's correct. Didn't think of that option. More the `SavePreferences.fromPreferences(JabRefPreferences)` version.